aws/README.md

```diff
@@ -1,9 +1,22 @@
 # AWS
 
-As of this writing, we support two types of AWS deployments with Cluster API:
-- A "standard" cluster with HA control plane and a machine deployment for workers
-- A cluster that uses autoscaling groups for the worker set.
-Autoscaling groups are called "MachinePools" in Cluster API lingo.
+As of this writing, we support two types of AWS deployments with Cluster API (CAPI):
+
+- A "standard" cluster with HA control plane and a machine deployment for workers.
+- A cluster that uses autoscaling groups for the worker set. Autoscaling groups are called "MachinePools" in Cluster API lingo.
+
+We also provide different variants for Talos 0.10 and up that added better support for
+[external cloud providers](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/),
+including [cloud-provider-aws](https://github.com/kubernetes/cloud-provider-aws).
+
+| Directory                                                       | Talos | cluster-api-provider-aws (CAPA) |
+| --------------------------------------------------------------- | ----- | ------------------------------- |
```

Comment:
should we do `v0.1/stantard/...` and `v0.2/standard/..` here saying that v0.2 is only for Talos v0.10+ with `cloudProvider: external` config

Reply:
The only reason I did not do that is my desire to avoid even more non-aligned version numbers… Will think about it.